torch_dftd/functions/edge_extraction.py

```diff
@@ -46,14 +110,10 @@ def calc_edge_index(
         if not bidirectional:
             raise NotImplementedError()
 
-        idx_i, idx_j, S = primitive_neighbor_list(
-            "ijS",
-            pbc.detach().cpu().numpy(),
-            cell.detach().cpu().numpy(),
-            pos.detach().cpu().numpy(),
-            cutoff,
-        )
-        edge_index = torch.tensor(np.stack([idx_i, idx_j], axis=0), device=pos.device)
-        # convert int64 -> pos.dtype (float)
-        S = torch.tensor(S, dtype=pos.dtype, device=pos.device)
+        try:
+            edge_index, S = calc_neighbor_by_pymatgen(pos, cell, pbc, cutoff)
+        except Exception:
```

Comment:
Can we limit the type of `Exception` here, e.g. `NotImplementedError`?

Reply:
I wanted to consider the other case than `NotImplementedError`, but I think it's nice to raise exit with error then. I will change it.